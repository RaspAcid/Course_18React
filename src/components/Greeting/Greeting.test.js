import renderer from 'react-test-renderer';
import { Greeting } from './Greeting';

test('renders greeting correctly', () => {
  jest.useFakeTimers();
  Date.now = jest.fn(() => 1789532048801);

  const tree = renderer.create(<Greeting />,).toJSON();
  expect(tree).toMatchSnapshot();
});