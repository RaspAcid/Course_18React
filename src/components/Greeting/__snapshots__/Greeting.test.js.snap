// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders greeting correctly 1`] = `
<div
  className="App"
>
  <p>
    Good 
    evening
    !
  </p>
</div>
`;
